cmd/api-server/main.go

```diff
@@ -325,6 +325,48 @@ func main() {
 		}
 		return notifications.Channel(), nil
 	}
+	getTestWorkflowServiceNotificationsStream := func(ctx context.Context, executionID, serviceName string, serviceIndex int) (<-chan testkube.TestWorkflowExecutionNotification, error) {
```

Comment:
I would think about extracting this to separate package - to not make main too big.

Reply:
side note: `main` will be split soon to smaller `main`s, so I guess it's not a big deal 🙂

Reply:
adressed